### PR TITLE
more succinct relative dates

### PR DESCRIPTION
**Pull request by mcginty**
Also opens us up to use relative dates ("3 mins", "now") in conversation items.

### CODE REVIEW

File: res/values/plurals.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<resources>
+    <plurals name="minutes_ago">
+        <item quantity="one">%d min</item>
+        <item quantity="other">%d mins</item>
+    </plurals>
+
+    <plurals name="hours_ago">
+        <item quantity="one">%d hour</item>
+        <item quantity="other">%d hours</item>
+    </plurals>
```
Comment (moxie0) [inline res/values/plurals.xml]: you don't have to localize this shit?

**Comment by mcginty** [comment]
Transifex claims to have native support for plurals

File: res/values/plurals.xml (hunk 1 above)
Comment (mcginty) [inline res/values/plurals.xml]: Transifex claims to have native support for plurals

**Comment by moxie0** [comment]
Transifex only knows about strings.xml, if these are in a different file TX won't even see them.

**Comment by mcginty** [comment]
Their documentation deceives! Moved to strings.

**Pull request merged**